code/datums/status_effects/debuffs/curse.dm

```diff
@@ -0,0 +1,202 @@
+#define SEVERE_THRESHOLD 6
+
+/**
+ * The cursed status effect.
+ * Increasingly worse effects are added as the heart is not pumped.
+ * See drunk.dm for reference.
+ */
+/datum/status_effect/cursed
+	id = "cursed"
+	tick_interval = 2 SECONDS
+	status_type = STATUS_EFFECT_REPLACE
+	var/curse_value = 0
+
+/datum/status_effect/cursed/on_creation(mob/living/new_owner, curse_value = 0)
+	. = ..()
+	set_curse_value(curse_value)
+
+/datum/status_effect/cursed/on_apply()
+	RegisterSignal(owner, COMSIG_LIVING_POST_FULLY_HEAL, .proc/clear_curse)
+	return TRUE
+
+/datum/status_effect/cursed/on_remove()
+	UnregisterSignal(owner, COMSIG_LIVING_POST_FULLY_HEAL)
+
+/datum/status_effect/cursed/get_examine_text()
+	// Dead people don't seem cursed
+	if(owner.stat == DEAD || HAS_TRAIT(owner, TRAIT_FAKEDEATH))
+		return null
+
+	switch(curse_value)
+		if(11 to 21)
+			return span_warning("[owner.p_they(TRUE)] [owner.p_are()] slightly pale.")
+		if(21.01 to 41)
```

Comment:
you dont need this .01 stuff, can just do from 21 to 41

Reply:
https://github.com/tgstation/tgstation/pull/68739/commits/e6fb9393dfc3cb3f2e5c0a1e43fbec85ba2f6066